src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class= "header">
+      <div class="header__image">
+        <a href="#" class="logo">
+          <img
+            src="./images/logo.png" alt="logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <div class="nav--links">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#laptop&computers"
+                class="nav__link"
+                data-qa="hover">laptops & computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="photo" class="nav__link">photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="video" class="nav__link">video</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#photo" class="nav__link">photo</a>
            </li>
            <li class="nav__item">
              <a href="#video" class="nav__link">video</a>
            </li>
```
6 point in checklist